packages/console/src/pages/GetStarted/index.tsx

```diff
@@ -18,9 +18,18 @@ const GetStarted = () => {
   const navigate = useNavigate();
   const { data, isLoading } = useGetStartedMetadata({ checkDemoAppExists: true });
   const { update } = useUserPreferences();
-  const [showConfirmModal, setShowConfirmModal] = useState(false);
 
-  const hideGetStarted = () => {
+  const hideGetStarted = async () => {
+    const confirmed = await confirmModalPromise({
+      content: t('get_started.confirm_message'),
+      confirmButtonType: 'primary',
+      confirmButtonText: 'admin_console.get_started.hide_this',
+    });
+
+    if (!confirmed) {
+      return;
+    }
```

Comment:
nit: looks like you use inline await above, `if(await(XXXX))`. Better consistent?  P.S. I prefer this one.

Reply:
updated.